trimesh/visual/material.py

```
@@ -338,7 +339,11 @@ def empty_material(color=None):
     material : SimpleMaterial
       Image is a a one pixel RGB
     """
-    from PIL import Image
+    try:
+        from PIL import Image
+    except BaseException as E:
+        return exceptions.ExceptionModule(E)
```

Should it be `raise` instead of `return`?

---

Sorry, read what is `ExceptionModule` about. Looks right.